mlflow/store/dynamodb_cfn.yaml

```diff
@@ -0,0 +1,221 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Parameters:
+  PrefixParameter:
+    Type: String
+    Default: mlflow
+    Description: Prefix for ddb tables
+  RCUParameter:
+    Type: Number
+    Default: 1
+    Description: Read Capacity Units for DynamoDb tables
+  WCUParameter:
+    Type: Number
+    Default: 1
+    Description: Write Capacity Units for DynamoDb tables
+Resources:
+  ExperimentTable:
+    Type: "AWS::DynamoDB::Table"
+    Properties:
+      AttributeDefinitions:
+        -
+          AttributeName: "experiment_id"
+          AttributeType: "N"
+        -
+          AttributeName: "lifecycle_stage"
+          AttributeType: "S"
+        -
+          AttributeName: "name"
+          AttributeType: "S"
+        # -
+        #   AttributeName: "artifact_location"
+        #   AttributeType: "S"
+        # -
+        #   AttributeName: "last_update_time"
+        #   AttributeType: "N"
+        # -
+        #   AttributeName: "creation_time"
+        #   AttributeType: "N"
+      KeySchema:
+        -
+          AttributeName: "experiment_id"
+          KeyType: "HASH"
+      ProvisionedThroughput:
+        ReadCapacityUnits: !Ref RCUParameter
+        WriteCapacityUnits: !Ref WCUParameter
+      GlobalSecondaryIndexes:
+        -
+          IndexName: LifeCycleStage
+          KeySchema:
+            -
+              AttributeName: "lifecycle_stage"
+              KeyType: "HASH"
+            -
+              AttributeName: "name"
+              KeyType: "RANGE"
+          Projection:
+            ProjectionType: "ALL"
+          ProvisionedThroughput:
+            ReadCapacityUnits: !Ref RCUParameter
+            WriteCapacityUnits: !Ref WCUParameter
+      TableName:
+        !Join
+          - '_'
+          - - !Ref PrefixParameter
+            - 'experiment'
+  RunTable:
+    Type: "AWS::DynamoDB::Table"
+    Properties:
+      AttributeDefinitions:
+        -
+          AttributeName: "run_uuid"
+          AttributeType: "S"
+        -
+          AttributeName: "experiment_id"
+          AttributeType: "N"
+        -
+          AttributeName: "lifecycle_stage"
+          AttributeType: "S"
+        # -
```

Comment:
Why are these commented out?

Reply:
Will remove, these NonKeyAttributes were included for reference initially.